core/server/web/shared/middlewares/trigger-webhooks.js

```diff
@@ -0,0 +1,13 @@
+const webhookService = require('../../../services/webhooks');
+const SITE_CHANGE_EVENT = 'site_changed';
```

Comment:
I think it would be good to be consistent with dot-notation for event names. So `site.changed` to be consistent with `subscriber.added` and `subscriber.deleted`

Reply:
👍

Reply:
Do we even need to declare a variable in case we use `.` notation (like [here](https://github.com/TryGhost/Ghost/pull/10019/files#diff-69440bce5a16e4d155ef391e67065795L50))? Seems rather conventional and having variables is just an extra mental load :thinking:

Reply:
We don't need to, was just following the style of extracting constants on top, pretty neutral about it here so we can just switch to direct usage. :)